### ISSUE TITLE
not possible to document fields in struct type error in error block.

### ISSUE THREAD

**Issue by andete** [brson/error-chain#97]
See [errors.rs](https://github.com/andete/philipshue/blob/use_error_chain/src/errors.rs) for an example.
As far as I can see it is not possible to add documentation to the 3 fields of BridgeError.

It would be nice if that could be made possible in some way.

**Comment by LFalch** [comment]
@andete The link for the example doesn't work anymore, because the branch is gone.

**Comment by andete** [comment]
alternative location for errors.rs: https://github.com/Orangenosecom/philipshue/blob/master/src/errors.rs

**Comment by andete** [comment]
Or rather more permanent: https://github.com/Orangenosecom/philipshue/blob/431d1b166e8a07baac60ea31f6820fb200b0149e/src/errors.rs

**Comment by Yamakaky** [comment]
^^
Unfortunately, https://github.com/brson/error-chain/issues/63 makes it more complicated that it should. Do you want to try?

**Comment by andete** [comment]
No, I just noticed the issue wasn't reported yet. Even though it seems to be related to #63 , for a casual user of error-chain this may not be obvious.

**Comment by Yamakaky** [comment]
Yeah, I myself didn't notice it before your issue ^^
